dt-reports/magic-url-bulk-send.php

```diff
@@ -14,12 +14,22 @@
     add_action( 'dt_post_bulk_list_link', 'dt_post_bulk_list_link_apps', 20, 3 );
     function dt_post_bulk_list_link_apps( $post_type, $post_settings, $dt_magic_apps ) {
         if ( ! empty( $dt_magic_apps ) && 'contacts' === $post_settings['post_type'] ) : ?>
-            <span style="display:inline-block">
-                <button class="button clear" id="bulk_send_app_controls" style="margin:0; padding:0">
-                    <?php esc_html_e( 'Bulk Send App', 'disciple_tools' ); ?>
-                    <img class="dt-icon" src="<?php echo esc_html( get_template_directory_uri() . '/dt-assets/images/connection.svg' ) ?>"/>
-                </button>
-            </span>
+            <script>
```

Comment:
@prykon can this section use the `dt_nav_dropdown_menu_items` filter?

Reply:
Done.